docs/intro.rst

```diff
@@ -0,0 +1,103 @@
+Camus
+================
+
+Camus is a pipeline from Kafka to HDFS.
```

Comment:
Agreed, this initial description needs to be very clear, concise, and do a great job of explaining what Camus can do for them. This means both explaining the core functionality ("capable of incrementally copying data...") and it's value (I think this is only half there with Neha's version, probably needs to also say something about making it easy to reliably load all your data from Kafka into Hadoop just by periodically running a Hadoop job).

A couple of notes:
* Give LinkedIn credit up front ("Camus is a simple MapReduce job developed by LinkedIn...").
* We need to describe how this integrates with the rest of the platform (something like "Confluent's version of Camus integrates with the schema-registry to automatically validate the data as it is loaded and ensure compatibility of data stored in HDFS as schemas are evolved.")
* I'd move the compatibility stuff to a subsection later on. It's not really important enough to put here. Maybe add a 'Requirements' subsection, which can discuss both Hadoop compatibility, mentioning CDH, and also the schema registry requirement.